ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -2,33 +2,36 @@
 import os
 import uuid
 
+from dcos_installer import backend
+
 
 class InstallerAnalytics():
-    def __init__(self, config):
+    def __init__(self):
         self.uuid = self.new_uuid()
-        self.bootstrap_version = os.getenv("BOOTSTRAP_ID")
-        self.customer_key = config["customer_key"] if 'customer_key' in config else ""
+        self.bootstrap_version = os.getenv("BOOTSTRAP_ID", "")
+        self.customer_key = ""
         self.source = "onprem"  # If you're using the installer you're not in AWS or Azure
 
-    def send(self, action, install_method, num_errors):
+    def send(self, action="", install_method="", num_errors=0):
```

Comment:
we never don't set all these parameters / use the defaults, so we aren't getting value out of having them. All they're doing is hiding programmer errors where we writing the code forget to set some of the parameters.